electron/launcher.ts

```diff
@@ -240,6 +240,29 @@ function setupEnvVars(gameSettings: GameSettings) {
   if (gameSettings.audioFix) {
     ret.PULSE_LATENCY_MSEC = '60'
   }
+  if (
+    !gameSettings.preferSystemLibs &&
+    gameSettings.wineVersion.type === 'wine'
+  ) {
+    if (gameSettings.wineVersion.bin) {
+      const lib32 = join(gameSettings.wineVersion.bin, '../../lib')
```

Comment:
another idea for dealing with that is adding the `lib` and `lib32` to the `wineVersion` object when getting the wine list. I think that would make it more flexible as well.
We also need to check the libraries for proton, or not? 🤔

Reply:
proton-ge is handling it by itself.If you open the proton executable in vscode you will see that is just a script.It sets the LD_LIBRARY_PATH in there.

Reply:
Actually, I agree. We need that same info in `runWineCommand`